extensions/amp-form/0.1/amp-form.js

```diff
@@ -72,13 +79,27 @@ export class AmpForm {
     }
     if (this.xhrAction_) {
       e.preventDefault();
+      this.setState_(FormState_.SUBMITTING);
```

Comment:
The only problem I see with applying this to non-xhr submissions are if the submission happened in `_blank` and then the user came back to the page the form would still likely be in a submitting state. This might make sense for `_top` though. What do you think?